### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by Ne-wt**
adds back dbal for new installs on older mariadb distributions, though this could possibly be removed / optional
I believe xampp covers it now though not certain

orderByRaw only needs to be changed to orderBy if DB::raw is present inside the query

### CODE REVIEW

**Comment by itinerare** [comment]
Probably safest to keep dbal for now given xampp, yeah… even if it’s supported at this point, easier to keep it and give folk time to update etc.

File: resources/views/admin/users/user.blade.php [hunk 1]
```diff
@@ -94,11 +95,9 @@
         @endif
         {!! Form::open(['url' => 'admin/users/' . $user->name . '/birthday']) !!}
         <div class="form-group row">
-            <label class="col-md-2 col-form-label">Birthday</label>
+            <label class="col-md-2 col-form-label">Date of Birth</label>
             <div class="col-md-10 row">
-                {{ Form::selectRange('dob[day]', 1, 31, $user->birthday ? intval($user->birthday->format('d')) : null, ['class' => 'form-control col-2 mr-1']) }}
-                {{ Form::selectMonth('dob[month]', $user->birthday ? intval($user->birthday->format('m')) : null, ['class' => 'form-control col-2 mr-1']) }}
-                {{ Form::selectYear('dob[year]', date('Y'), date('Y') - 70, $user->birthday ? intval($user->birthday->format('Y')) : null, ['class' => 'form-control col-2']) }}
+                {!! Form::date('dob', null, ['class' => 'form-control']) !!}
```
Comment (itinerare) [inline resources/views/admin/users/user.blade.php:100]: Oh good, I realized this spot probably hadn’t gotten caught earlier but forgot to mention it. |D

**Review by itinerare — APPROVED** [review]

**Comment by itinerare** [comment]
Oh yes, and it’s not much trouble atm since it’s just dependency updates/the release branch is very fresh anyway, but remember to base stuff on the release branch for whenever the next round of fixes is.

**Comment by Ne-wt** [comment]
> Oh yes, and it’s not much trouble atm since it’s just dependency updates/the release branch is very fresh anyway, but remember to base stuff on the release branch for whenever the next round of fixes is.

yeah I realised that after I saw all the dependency updates lol

**Review by SpeedyD — APPROVED** [review]

**Pull request merged**